spec/controllers/downloads_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe DownloadsController do
+  describe '#show' do
+    let(:user) { FactoryGirl.create(:admin) }
+    let(:file_set) do
+      FactoryGirl.create(:file_set, user: user, content: File.open(fixture_path + '/files/video.mp4'))
+    end
+
+    context "when the user has access" do
+      before { sign_in user }
+
+      it 'sends the original file' do
+        get :show, :id => file_set.id
+        expect(response.body).to eq file_set.original_file.content
+      end
+
+      it 'sends the derivative' do
```

Comment:
@mcritchlow Yep, it looks like the FileSets working group is trying to provide support to ingest multiple files to the Fileset with different use in the UI level as well. We will know what we can have from upstream soon and see what we need to do next.